src/main/java/cz/xtf/radanalytics/oshinko/deployment/Oshinko.java

```diff
@@ -116,13 +119,13 @@ private static OshinkoPoddedWebUI deployWebUIPodCommonLogic(String templateName,
 	 *
 	 * @return path to Oshinko Webui resources yaml file
 	 */
-	private static String getOshinkoWebuiResources(String oshinkoWebUITemplate) {
-		log.info("Getting Oshinko Web UI Resources from temp directory");
+	public static String getOshinkoResources(String workDir, String oshinkoTemplate, String oshinko_resources_url) {
+		log.info("Getting Oshinko Resources from temp directory");
 
 		if (OSHINKO_WEBUI_RESOURCES == null) {
```

Comment:
but would be great to keep this functionality, ie. to download each resource only once. Please try to keep this in mind while refactoring. Thanks!